client/components/theme/test/__snapshots__/index.jsx.snap

```diff
@@ -30,9 +30,6 @@ exports[`Theme rendering with default display buttonContents should match snapsh
       >
         Twenty Seventeen
       </h2>
-      <span
-        class="theme__badge-price theme__badge-price-upgrade"
```

Comment:
Below is the original code before we remove the feature flag. However, the value is false because we don't turn on the flag in the test environment. Thus, the price will show in snapshots.
```js
{ ( ! isEnabled( 'signup/seller-upgrade-modal' ) || active ) && (
  <span className={ priceClass }>{ price }</span>
) }
```

Reply:
Ah okay! I thought it would still show up since the test was changed:
```diff
-const { container } = render( <Theme { ...props } price="$50" /> );
+const { container } = render( <Theme { ...props } price="$50" active /> );
```

The price would still show up since:
```
{ active && <span className={ priceClass }>{ price }</span> }
```

Maybe I misunderstood, and the snapshot is not related to the test?

Reply:
The snapshot is generated from [here](https://github.com/Automattic/wp-calypso/blob/4260dcad4429914e261dad0610e16fcd028a937e/client/components/theme/test/index.jsx#L78)

```js
test( 'should match snapshot', () => {
  const { container } = render( <Theme { ...props } /> );
  expect( container.firstChild ).toMatchSnapshot();
} );
```